sequencing-server/src/lib/codegen/CommandTypeCodegen.ts

```diff
@@ -17,23 +17,39 @@ function generateTypescriptCode(dictionary: ampcs.CommandDictionary): {
     typescriptFswCommands.push(generateFswCommandCode(fswCommand, dictionary.enumMap));
   }
 
+  const typescriptHwCommands: { value: string; interfaces: string }[] = [];
+  for (const hwCommand of dictionary.hwCommands) {
+    typescriptHwCommands.push(generateHwCommandCode(hwCommand));
+  }
+
   // language=TypeScript
   const declarations = `
 declare global {
-${typescriptFswCommands.map(fswCommand => fswCommand.interfaces).join('\n')}
+${typescriptFswCommands.map(fswCommand => fswCommand.interfaces).join('\n')}${typescriptHwCommands
+    .map(hwCommand => hwCommand.interfaces)
+    .join('\n')}\n
 \tconst Commands: {\n${dictionary.fswCommands
     .map(fswCommand => `\t\t${fswCommand.stem}: typeof ${fswCommand.stem},\n`)
     .join('')}\t};
+
+\tconst Hwds : {\n${dictionary.hwCommands
+    .map(hwCommand => `\t\t${hwCommand.stem}: typeof ${hwCommand.stem}`)
+    .join(',\n')} \t};
 }`;
 
   // language=TypeScript
   const values = `
 ${typescriptFswCommands.map(fswCommand => fswCommand.value).join('\n')}
+${typescriptHwCommands.map(hwCommands => hwCommands.value).join('\n')}\n
 export const Commands = {${dictionary.fswCommands
     .map(fswCommand => `\t\t${fswCommand.stem}: ${fswCommand.stem},\n`)
     .join('')}};
 
-Object.assign(globalThis, { A:A, R:R, E:E, C:Object.assign(Commands, STEPS), Sequence});
+export const Hwds = {${dictionary.hwCommands
```

Comment:
Any reason this isn't fully spelt out like `Commands` or `Sequence`?

Reply:
No reason just shorthand for Hardware. Let me know if you want to change it.

Reply:
Sounds good, I made the change